### ISSUE TITLE
interface.system_table test is failing

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#4403]
```
    self.test.run()
  File "/home/ssd3/daniel/rethinkdb/test/common/shelltest.py", line 31, in run
    check_call(self.command, shell=True, env=env)
  File "/usr/lib/python2.7/subprocess.py", line 511, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command '$RETHINKDB/test/interface/system_changefeeds.py' returned non-zero exit status 1


Traceback (most recent call last):
  File "/home/ssd3/daniel/rethinkdb/test/interface/system_changefeeds.py", line 122, in <module>
    "test_config", "test_status", "test2_config", "test2_status"], 1.0)
  File "/home/ssd3/daniel/rethinkdb/test/interface/system_changefeeds.py", line 75, in check
    "Expected changes on %s, found none." % name
AssertionError: Expected changes on test_config, found none.
Command '$RETHINKDB/test/interface/system_changefeeds.py' returned non-zero exit status 1
```

This is because the table config isn't updated when a server is renamed. @VeXocide is looking into it.

**Comment by VeXocide** [comment]
This will be fixed as part of #4252.

**Comment by VeXocide** [comment]
After fixing #4252 it still fails, but with a new error:

```
FAIL interface.system_changefeeds
    self.test.run()
  File "/home/ssd3/vexocide/rethinkdb/test/common/shelltest.py", line 31, in run
    check_call(self.command, shell=True, env=env)
  File "/usr/lib/python2.7/subprocess.py", line 511, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command '$RETHINKDB/test/interface/system_changefeeds.py' returned non-zero exit status 1


Traceback (most recent call last):
  File "/home/ssd3/vexocide/rethinkdb/test/interface/system_changefeeds.py", line 127, in <module>
    "test_status", "test2_status"], 1.0)
  File "/home/ssd3/vexocide/rethinkdb/test/interface/system_changefeeds.py", line 79, in check
    "Expected no changes on %s, found %s." % (name, feed.changes)
AssertionError: Expected no changes on table_config, found [{u'old_val': {u'primary_key': u'id', u'durability': u'hard', u'db': u'test', u'shards': [{u'nonvoting_replicas': [], u'primary_replica': u'c', u'replicas': [u'c']}], u'write_acks': u'majority', u'id': u'437fc34d-29ee-424a-ba06-e515aedbee41', u'name': u'test2'}, u'new_val': {u'db': u'test', u'id': u'437fc34d-29ee-424a-ba06-e515aedbee41', u'name': u'test2', u'error': u'None of the replicas for this table are accessible.'}}].
Command '$RETHINKDB/test/interface/system_changefeeds.py' returned non-zero exit status 1
```

**Comment by VeXocide** [comment]
Merged into `next` via commit 86a7f72f83ec7f7297e0abaee7df2b5b51f812e8, over-the-shoulder by @danielmewes.
